designs/developer/4.8/quickstarts-quick-add/index.md

```diff
@@ -0,0 +1,34 @@
+---
+parent: Developer
+version: 4.8
+---
+
+# Quick Starts from the Quick Add menu
+
+## General search 
+
+![Quick Starts from the quick add menu](img/1-quickstarts-quick-add-menu.png)
+
+ If the user searches by a specific term and there are quick starts available for that term, the Quick Start appears as an option. Quick Starts now appear along side the Developer Catalog items if it matches the search term.
+
+![Start Quick Start](img/2-start-quickstart-from-menu.png)
+
+The user can start the quick start by clicking the ‘Start’ button.
+
+![Quick Start opens](img/3-Open-quickstart.png)
+
+The selected quick start starts in context of the Topology.
+
+![View all Quick Starts](img/4-View-all-quickstarts.png)
+
+The user can also click on the view all results in the Quick Starts catalog to see their search results there.
+
+![Quick Starts Catalog](img/5-Quickstart-catalog.png)
+
+The user is taken to the search results in context of the Quick Starts Catalog.
```

Comment:
Lower case quick starts catalog?

Reply:
Can do, I'll make quick starts lowercase in the places that you noted.